### PR TITLE
ansible: replace DO ubuntu1604-x86 hosts with ubuntu2204-x64

### PR DESCRIPTION
**Pull request by targos**
Refs: https://github.com/nodejs/build/issues/3317

### CODE REVIEW

**Comment by targos** [comment]
This is still work-in-progress.

File: ansible/roles/baselayout/vars/main.yml [hunk 1]
```diff
@@ -143,7 +143,8 @@ packages: {
     'gcc-6,g++-6,gcc-8,g++-8,python3.8',
   ],  
 
+  # Default gcc/g++ package is 11.
   ubuntu2204: [
-    'gcc,g++,python2,python3,python-is-python3',
+    'gcc,g++,python3,python-is-python3',
```
Comment (targos) [inline ansible/roles/baselayout/vars/main.yml:148]: We shouldn't need python2 anymore, right?

**Comment by targos** [comment]
Hosts are online:
https://ci.nodejs.org/computer/test%2Ddigitalocean%2Dubuntu2204%2Dx64%2D1/
https://ci.nodejs.org/computer/test%2Ddigitalocean%2Dubuntu2204%2Dx64%2D2/

File: ansible/roles/baselayout/vars/main.yml (hunk 1 above)
Comment (richardlau) [reply ansible/roles/baselayout/vars/main.yml:148]: Correct. In theory just Python 3 should work. I'm not even sure we need `python-is-python3`.

**Comment by targos** [comment]
Jobs are green!

**Comment by ovflowd** [comment]
> Jobs are green!

🎉 

What are the next steps?

**Review by ovflowd — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Pull request merged**